### DISCUSSION TITLE
Can we get more uploading speed?

### DISCUSSION THREAD

**Discussion by DKB0512** [kotatogram/kotatogram-desktop#191] [EN 2. Q&A]
I tried that parallel uploading with my Gigabit connection and the max I am getting is around 100Mbps for uploading, I want to know that can we get more than that? I have selected Big for parallel uploads tho and still getting only around 100Mbps or 5Mb/sec.

**Comment by ilya-fedin** [comment]
Maybe Telegram servers can't give more? :thinking: 
Or some other point on the route to them...

**Reply by DKB0512** [reply]
I have seen people using this : https://gist.github.com/painor/7e74de80ae0c819d3e9abcf9989a8dd6
and getting around 600Mbps during uploading
![image](https://user-images.githubusercontent.com/23116307/126299714-cfb0add8-b3f2-47a5-83a1-536eb680f760.png)

Here is the link to that post : https://github.com/iw4p/telegram-cloud/issues/25#issuecomment-851953773
